src/dsp_tools/utils/excel2json/lists.py

```diff
@@ -272,13 +272,13 @@ def validate_lists_section_with_schema(
         lists_section: the "lists" section as Python object
 
     Raises:
-        BaseError: if the validation fails
+        UserError: if the validation fails
 
     Returns:
         True if the "lists" section passed validation
     """
     if bool(path_to_json_project_file) == bool(lists_section):
-        raise BaseError("Validation of the 'lists' section works only if exactly one of the two arguments is given.")
+        raise UserError("Validation of the 'lists' section works only if exactly one of the two arguments is given.")
```

Comment:
This should be a base error. Because it can only occur if a developer has used the function in a wrong way. It is not possible that a user will ever get this error

Reply:
It doesn't need to be handled, because if the code of DSP-TOOLS was written correctly, it will never occur. It can only occur if we ourselves make a programming mistake and call this function with both arguments.